### ISSUE TITLE
Syntax error with grouped use statement and multiple use with comma

### ISSUE THREAD

**Issue by pierredup** [PHP-CS-Fixer/PHP-CS-Fixer#8165]
### Problem description

When combining grouped use statement and a use statement using commas, I get a syntax error when running PHP-CS-Fixer.

```
Found 0 of 1 files that can be fixed in 0.002 seconds, 16.00 MB memory used

Files that were not fixed due to errors reported during linting after fixing:
   1) test.php


        [PhpCsFixer\Linter\LintingException]
        Parse error: syntax error, unexpected token ",", expecting ";" on line 4.


      Applied fixers: ordered_imports
```

The error specifically mentions the line with use statements separated by a comma (which in itself is fine and not a syntax issue), but it only breaks with the precise semantics of the script below and using the `ordered_imports` rule

### Minimal reproducer

```bash
php-cs-fixer check test.php -vvv --rules ordered_imports
```

```php
<?php

use Doc\{A};
use DateTime, Exception;

```

### Fixer version

3.59.3

### PHP version

8.1.29

### How do you run Fixer?

Composer Shim package (binary only)

### Contribution Checks

- [x] I have verified if this problem was already reported
- [X] I am familiar with ["Feature or bug?"](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/feature-or-bug.rst)

**Comment by kubawerlos** [comment]
Indeed, it is a bug, `OrderedImportsFixer` tries to change your example to:
```php
<?php

use DateTime;
use Doc\{A}, Exception;
```

**Comment by theodorejb** [comment]
This is still an issue.

**Comment by theodorejb** [comment]
This is still an issue in PHP-CS-Fixer 3.68.5.
